docs/rapha-rover/specification/ros-api.mdx

```diff
@@ -0,0 +1,567 @@
+
+  Calibration data for the center (RGB) Oak camera.
+
+- `oak/rgb/image_raw` ([sensor_msgs/msg/Image])
+
+  Raw images from the center (RGB) Oak camera.
+
+- `oak/right/camera_info` ([sensor_msgs/msg/CameraInfo])
+
+  Calibration data for the right Oak camera.
+
+- `oak/right/image_raw` ([sensor_msgs/msg/Image])
+
+  Raw images from the right Oak camera.
+
+- `oak/stereo/camera_info` ([sensor_msgs/msg/CameraInfo])
+
+  Calibration data for **TODO**
+
+- `oak/stereo/image_raw` ([sensor_msgs/msg/Image])
+
+  Depth images calculated from the stereo pair (left and right Oak cameras).
+
+- `oak/points` ([sensor_msgs/msg/PointCloud2])
+
+  Point cloud data converted from the depth images.
+
+- `rplidar/scan` ([sensor_msgs/msg/LaserScan])
+
+  Raw laser scans from the onboard 2D LIDAR.
+
+- `rplidar/scan_filtered` ([sensor_msgs/msg/LaserScan]) **Not implemented yet**
+
+  Laser scans from the onboard 2D LIDAR with points lying on the robots
+  footprint filtered out.
+
+- `joint_states` ([sensor_msgs/msg/JointState])
+
+  Current state of the robot joints.
+
+- `robot_description` ([std_msgs/msg/String])
+
+  The URDF description of the robot.
+
+- `/diagnostics` ([diagnostic_msgs/msg/DiagnosticArray])
+
+  Diagnostic information from various ROS nodes.
+
+- `/parameter_events` ([rcl_interfaces/msg/ParameterEvent])
+
+  Parameter events (additions, changes or deletions) from all the running ROS
+  nodes.
+
+- `/rosout` ([rcl_interfaces/msg/Log])
+
+  Logs from all the running ROS nodes.
+
+- `/tf`, `/tf_static` ([tf2_msgs/msg/TFMessage])
+
+  Relationships between TF frames in the ROS network. \
+  These topics are used by the
+  [tf2 library](https://docs.ros.org/en/jazzy/Concepts/Intermediate/About-Tf2.html).
+
+## Services
+
+- `controller/clear_leds` ([std_srvs/srv/Trigger])
+
+  Resets user states for all LEDs in the LED strip.
+
+- `controller/set_leds` ([rapha_interfaces/srv/SetLeds])
+
+  Set user state for specified LEDs.
+
+- `controller/calibrate_servos` ([std_srvs/srv/Trigger])
+
+  Trigger servo calibration procedure.
+
+- `controller/reset_odom` ([std_srvs/srv/Trigger])
+
+  Reset odometry position.
+
+- `controller/reset_motor_diagnostics` ([std_srvs/srv/Trigger])
+
+  Reset motor faults and calculated consumed energy.
+
+- `controller/set_battery_modes` ([rapha_interfaces/srv/SetBatteryModes])
+
+  Set target battery modes.
+
+- `controller/set_steering_mode` ([rapha_interfaces/srv/SetSteeringMode])
+
+  Set the current steering mode.
+
+## Parameters
+
+### controller
+
+- `drivetrain.wheel_base` (type: `float`, default: `0.378`)
+
+  Distance between front and rear wheels. [m]
+
+- `drivetrain.track_width` (type: `float`, default: `0.384`)
+
+  Distance between left and right wheels. [m]
+
+- `drivetrain.wheel_radius` (type: `float`, default: `0.08`)
+
+  Radius of the wheel. [m]
+
+- `drivetrain.max_wheel_angular_velocity` (type: `float`, default: `19.0`)
+
+  Maximum allowable wheel angular velocity. [rad/s]
```

Comment:
Maybe we can use KaTeX for specifing units?
```suggestion
  Maximum allowable wheel angular velocity. $\frac{rad}{s}$
```

Reply:
done